assessment.md

```diff
@@ -119,56 +119,57 @@ when creating python programs.
   * E = Issue is unique, well described, and a possible reason for occurrence
     and solution is mentioned
 * Using the GitHub flow model
-  * N = GitHub flow model unused or no commits
-  * I = Pushed to master or non-descriptive commits
-  * A = Pushed to correct repository with descriptive commits
+  * N = GitHub flow model unused or no commits to the Github repository
+  * I = Pushed to master or non-descriptive commits to the Github repository
+  * A = Pushed to correct repository with descriptive commits to the Github
```

Comment:
This repeats the word repository unnecessarily, and should be reworded. There shouldn't be a need to append `to the repository` on the end of each of these sentences.

Reply:
Does it look better now?

Reply:
The line that was the problem is line 124 and 125 -- however this change is good as well!